devops/openshift/database/patroni-deploy.yml

```diff
@@ -182,6 +182,17 @@ objects:
         resources:
           requests:
             storage: ${PVC_SIZE}
+- apiVersion: policy/v1
+  kind: PodDisruptionBudget
+  metadata:
```

Comment:
How we would apply those changes to PROD? Is there a plan for it?

Reply:
The idea is to deploy the PDB configuration in our namespace manually, than running the yaml.